src/lib/holocene/input/chip-input.svelte

```diff
@@ -163,6 +185,14 @@
     @apply surface-primary inline-block w-full focus:outline-none;
   }
 
+  .unroundRight {
```

Comment:
Could we just do something like `first-child:` or whatever?

Reply:
I looked into this and it was it not as simple as that (it's not the first child for one thing), but if we can figure out how to get this working with modifiers I'll update this for all our input components.